src/filters.ts

```diff
@@ -0,0 +1,244 @@
+import type {
+    ChatMember,
+    ChatMemberAdministrator,
+    ChatMemberBanned,
+    ChatMemberLeft,
+    ChatMemberMember,
+    ChatMemberOwner,
+    ChatMemberRestricted,
+    Context,
+    Filter,
+} from "./deps.deno.ts";
+
+/**
```

Comment:
Not a TSDoc comment, just an implementation notice
```suggestion
/*
```